proto/src/determined/api/v1/config_policies.proto

```diff
@@ -45,9 +57,14 @@ message PutGlobalConfigPoliciesResponse {
 // GetWorkspaceConfigPoliciesRequest lists task config policies
 // for a given workspace and workload type.
 message GetWorkspaceConfigPoliciesRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "workload_type" ] }
```

Comment:
is `workspace_id` also required here?

Reply:
Great suggestion; I'll add it to all the workspace requests.